scripts/jenkins-yoctobuild-init-script.sh

```diff
@@ -71,5 +71,7 @@ pip3 install pyyaml --upgrade
 # use sar from sysstat to render the result file (~/sysstat.log) manually
 apt_get -qy --force-yes install sysstat
 sudo sed -i 's/false/true/g' /etc/default/sysstat
+sudo rm -f /etc/cron.d/sysstat || true # see QA-223 just in case sa1 from cron
+                                       # conflicts with the sar below
```

Comment:
@kacf This file is not in use any more, right? Should we just remove it?

Reply:
I think it still may be used by the GCP Jenkins slave builder, which is only launched manually, but is nevertheless required for Jenkins to run. Not 100% sure though.

Will you ever need Jenkins anymore? It can definitely be killed when Jenkins receives the final R.I.P.

Reply:
@kacf Thanks, I didn't think about that. I'll have it in mind then.